2021-12-24 13:59:58,816 INFO    MainThread:3521 [wandb_setup.py:_flush():69] setting env: {}
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/tagger_text/wandb/run-20211224_135958-ctsyn24d/logs/debug.log
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/tagger_text/wandb/run-20211224_135958-ctsyn24d/logs/debug-internal.log
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_init.py:init():375] calling init triggers
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/tags_text/tags_post_texts_v5.csv', 'checkpoint_path': '/root/konst/model_checkpoints/tagger_text', 'model_path': '', 'batch_size': 128, 'epochs': 200, 'start_epoch': 0, 'test_size': 0.2, 'lam': 0, 'patience': 8, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 0}
2021-12-24 13:59:58,817 INFO    MainThread:3521 [wandb_init.py:init():424] starting backend
2021-12-24 13:59:58,818 INFO    MainThread:3521 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-24 13:59:58,829 INFO    MainThread:3521 [backend.py:ensure_launched():135] starting backend process...
2021-12-24 13:59:58,834 INFO    MainThread:3521 [backend.py:ensure_launched():140] started backend process with pid: 3546
2021-12-24 13:59:58,835 INFO    MainThread:3521 [wandb_init.py:init():429] backend started and connected
2021-12-24 13:59:58,838 INFO    MainThread:3521 [wandb_init.py:init():477] updated telemetry
2021-12-24 13:59:58,839 INFO    MainThread:3521 [wandb_init.py:init():500] communicating current version
2021-12-24 13:59:59,363 INFO    MainThread:3521 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-24 13:59:59,363 INFO    MainThread:3521 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-12-24 13:59:59,587 INFO    MainThread:3521 [wandb_init.py:init():540] starting run threads in backend
2021-12-24 14:00:00,348 INFO    MainThread:3521 [wandb_run.py:_console_start():1584] atexit reg
2021-12-24 14:00:00,348 INFO    MainThread:3521 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-12-24 14:00:00,349 INFO    MainThread:3521 [wandb_run.py:_redirect():1463] Redirecting console.
2021-12-24 14:00:00,350 INFO    MainThread:3521 [wandb_run.py:_redirect():1519] Redirects installed.
2021-12-24 14:00:00,350 INFO    MainThread:3521 [wandb_init.py:init():565] run started, returning control to user process
2021-12-24 14:05:13,863 INFO    MainThread:3521 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-12-24 14:05:13,864 INFO    MainThread:3521 [wandb_run.py:_restore():1526] restore
2021-12-24 14:05:15,888 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31867
  total_bytes: 31867
}

2021-12-24 14:05:16,060 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31867
  total_bytes: 31867
}

2021-12-24 14:05:16,690 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 4
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31867
  total_bytes: 40703
}

2021-12-24 14:05:16,792 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31867
  total_bytes: 40731
}

2021-12-24 14:05:16,894 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 38573
  total_bytes: 40731
}

2021-12-24 14:05:16,997 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 40731
  total_bytes: 40731
}

2021-12-24 14:05:17,101 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 40731
  total_bytes: 40731
}

2021-12-24 14:05:17,204 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 40731
  total_bytes: 40731
}

2021-12-24 14:05:17,307 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 40731
  total_bytes: 40731
}

2021-12-24 14:05:17,902 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 40731
  total_bytes: 40731
}

2021-12-24 14:05:18,005 INFO    MainThread:3521 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 40731
  total_bytes: 40731
}

